crmsh/bootstrap.py

```diff
@@ -2159,105 +2160,42 @@ def start_qdevice_on_join_node(seed_host):
     status_done()
 
 
-def remove_ssh():
-    seed_host = _context.cluster_node
-    if not seed_host:
-        error("No existing IP/hostname specified (use -c option)")
-
-    # check whether corosync has been started on local node
-    if not utils.service_is_active("corosync.service"):
-        error("Cluster is not active - can't execute removing action")
-
-    remove_get_hostname(seed_host)
-
-
-def remove_get_hostname(seed_host):
+def get_cluster_node_ip():
     """
-    Get the nodename
+    ringx_addr might be hostname or IP
+    _context.cluster_node by now is always hostname
 
-    Sets _context.host_status to
-    0 = Disconnected
-    1 = Connected
-    2 = Connected through IP address
+    If ring0_addr is IP, we should get the configured iplist which belong _context.cluster_node
+    Then filter out which one is configured as ring0_addr
+    At last assign that ip to _context.cluster_node_ip which will be removed later
     """
-    _context.connect_name = seed_host
-    _context.host_status = 0
-
-    # if node is localhost, assume connected and hostname = cluster name
-    if seed_host == utils.this_node():
-        _context.cluster_node = seed_host
-        _context.host_status = 1
+    node = _context.cluster_node
+    addr_list = corosync.get_values('nodelist.node.ring0_addr')
+    if node in addr_list:
```

Comment:
Why don't you just assign node to `cluster_node`? This way you don't need to do more checks in the future code (like `del_target = _context.cluster_node_ip or node`